src/PrestaShopBundle/DependencyInjection/Compiler/ModulesDoctrineCompilerPass.php

```diff
@@ -111,7 +111,10 @@ private function createAnnotationMappingDriver($moduleNamespace, $moduleEntityDi
     {
         $reader = new Reference('annotation_reader');
         $driverDefinition = new Definition('Doctrine\ORM\Mapping\Driver\AnnotationDriver', [$reader, [$moduleEntityDirectory]]);
-        $driverDefinition->addMethodCall('addExcludePaths', [[$moduleEntityDirectory . '/index.php']]);
+        $indexFile = $moduleEntityDirectory . '/index.php';
+        if (file_exists($indexFile)) {
+            $driverDefinition->addMethodCall('addExcludePaths', [[$moduleEntityDirectory . '/index.php']]);
```

Comment:
Why not use `$indexFile` instead of `$moduleEntityDirectory . '/index.php'`?

Reply:
You're right, that's what I intended to do!! ^^
Thank you